app/src/main/java/com/framgia/soundclound/screen/playlist/OnItemAlbumClick.java

```diff
@@ -0,0 +1,15 @@
+package com.framgia.soundclound.screen.playlist;
+
+import android.view.View;
```

Comment:
ERROR: Unused import: android.view.View. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports